### DISCUSSION TITLE
Hardware and software requirement for server

### DISCUSSION THREAD

**Discussion by harshakhmk** [qdrant/qdrant#3279] [General]
I am exploring Qdrant and tried out few examples, but was wondering what were the Hardware and software requirement for running and building the server

**Comment by generall** [comment]
You can run qdrant on as small as 0.5 CPU machine / 1Gb ram. Of course, the amount of data would be the main criteria of for choosing a specific hardware.

For building, it should be enough to have a machine with about 4Gb of RAM and about 40Gb hard drive

**Reply by harshakhmk** [reply]
For calculating the amount of CPU, RAM: htop would give the details 
About the storage required for data to be inserted, is there any accurate way to measure?
Also htop is not reliable as mentioned in this [article](https://qdrant.tech/articles/memory-consumption/), is there any alternative to measure RAM, CPU storage while running Qdrant as a standalone application
